README.md

```diff
@@ -45,3 +45,32 @@ Create a kubernetes manifest for a pod which will containa ToDo app container:
 1. There should not be any requests failing with 404 status code in browser console.
 1. `README.md` should have instructuions on how to validate the changes
 1. Create PR with your changes and attach it for validation on a platform.
+
+
+
+##### how to validate the changes
+
+1. Run manifests
+
+kubectl apply -f .infrastructure/app
+kubectl apply -f .infrastructure/mysql
```

Comment:
Use bootstrap.sh instead, but fix it(bootstrap.sh) before usage.

Reply:
Changed.